client/src/lib/votecounting.test.ts

```diff
@@ -0,0 +1,73 @@
+import { Candidate, Election } from '@votingworks/ballot-encoder'
+
+import * as path from 'path'
+import { promises as fs } from 'fs'
+
+import find from '../utils/find'
+import { parseCVRs, fullTallyVotes } from './votecounting'
+
+const fixturesPath = path.join(__dirname, '../../test/fixtures')
+const electionFilePath = path.join(fixturesPath, 'election.json')
+const cvrFilePath = path.join(fixturesPath, 'CVRs.txt')
+
+test('tabulating a set of CVRs gives expected output', async () => {
+  // get the election
+  const election = JSON.parse(
+    (await fs.readFile(electionFilePath)).toString('utf-8')
+  ) as Election
```

Comment:
Is the following simpler than using `fs` to read the file?

```
import electionFixture from '../../test/fixtures/election.json`?
const electionDefinition = electionFixture as Election
```

Reply:
I know we've done that before, but I prefer the more explicit loading, especially since we have to do that for CVRs, so doing it differently in the same file just seems weird.